tests/integration/cassettes/TestLiveThread_test_init.json

```diff
@@ -0,0 +1,164 @@
+{
+  "http_interactions": [
+    {
+      "recorded_at": "2016-12-06T14:54:15",
+      "request": {
+        "body": {
+          "encoding": "utf-8",
+          "string": "grant_type=client_credentials"
+        },
+        "headers": {
+          "Accept": "*/*",
+          "Accept-Encoding": "identity",
+          "Authorization": "Basic <BASIC_AUTH>",
+          "Connection": "keep-alive",
+          "Content-Length": "29",
+          "Content-Type": "application/x-www-form-urlencoded",
+          "User-Agent": "<USER_AGENT> PRAW/4.0.0 prawcore/0.3.0"
+        },
+        "method": "POST",
+        "uri": "https://www.reddit.com/api/v1/access_token"
+      },
+      "response": {
+        "body": {
+          "encoding": "UTF-8",
+          "string": "{\"access_token\": \"tuIhJlqr1dLVDqCFbVqtWF7tynw\", \"token_type\": \"bearer\", \"expires_in\": 3600, \"scope\": \"*\"}"
```

Comment:
I was thinking about the access token again. Given that this access token requires both the client id and client secret, it's effectively unusable without those even within the 1-hour timeframe.

Of course, I'm still open to a PR that removes the access token just to save people from the thought that they might be able to use the tokens.